packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -129,6 +129,7 @@ export function TransferPanel() {
       isTeleportMode
     }
   } = useLatest(useNetworksRelationship(latestNetworks.current))
+  const isDepositOrTeleportMode = isDepositMode || isTeleportMode
```

Comment:
We could add `isDepositOrTeleportMode`  to `useNetworksRelationships` and return from there